syncto/views/record.py

```diff
@@ -30,3 +47,26 @@ def record_get(request):
     handle_headers_conversion(sync_client.raw_resp, request.response)
 
     return {'data': record}
+
+
+@record.put(permission=NO_PERMISSION_REQUIRED, schema=PayloadSchema)
+def record_put(request):
+    collection_name = request.matchdict['collection_name']
+    record_id = request.matchdict['record_id']
+    sync_id = uuid4_to_base64(record_id)
+
+    if_unmodified_since = request.headers.get('If-Match')
+
+    record = request.validated['data']
+    record['id'] = sync_id
+
+    sync_client = build_sync_client(request)
+    last_modified = sync_client.put_record(collection_name, record,
```

Comment:
I have the same concerns as for the other pull request (about DELETE I think): we should try/catch any error that arise here, otherwise the server might just blow up if the request fails.